include/memory/memory.h

```diff
@@ -1,6 +1,14 @@
 #ifndef MEMORY_H_
 #define MEMORY_H_
 
-int memory_flash_region(const void *vAddress, const void *vData, unsigned int length);
+typedef enum {
```

Comment:
Does this mean we are keeping the typedef?  Or can you fix that?  If fixable then just go that way and be done with it.